### DISCUSSION TITLE
Bounty Hunter and Logistics Expert don't give extra money for completing missions

### DISCUSSION THREAD

**Discussion by StranDead1** [FakeFishGames/Barotrauma#14310] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Talents in multiplayer don't work. Specifically, talents for extra money for completing missions. 
These are the talents of the captain: Bounty Hunter, Logistics Expert
Additional money is not allocated, although skills for additional reputation work, additional reputation is added.
The "Figurehead" skill - I did not check

USING MODS: 
Sonar Extension
Super Mining Sonar
(I use two because one breaks the number of items in one slot - the other fixes)

### Reproduction steps

1. Start the multiplayer server for the captain
2. Take the skills of Bounty Hunter, Logistics Expert
3. Complete the relevant tasks
4. Finish the mission by arriving at the outpost

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
I can confirm that in multiplayer v1.5.9.1 talents Bounty Hunter, Logistics Expert do not show you that you got extra money on final screen, but in fact extra money is added to the bank

**Reply by StranDead1** [reply]
thanks

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in our private repo now, fix coming up in the next update. Sorry for the inconvenience!
